task15/hw15_enum.py

```diff
@@ -0,0 +1,56 @@
+# Enum
+
+from enum import Enum
+
+
+class OrderStatus(Enum):
+    PENDING = "Заказ ожидает обработки"
+    IN_PROGRESS = "Заказ готовится"
+    READY = "Заказ готов"
+    COMPLETED = "Заказ выдан"
+    CANCELLED = "Заказ отменён"
+
+
+class Order:
+    def __init__(self, order_id, status=OrderStatus.PENDING):
+        try:
+            self.order_id = int(order_id)
+            if self.order_id <= 0:
+                raise ValueError
+        except (ValueError, TypeError):
+            print("Ошибка: Неверный ID заказа. ID должен быть положительным числом.")
+            self.order_id = None
+            self.status = None
+            return
```

Comment:
Try someing like this:
```suggestion
    self.order_id = int(order_id) if int(order_id) > 0 else None
    self.status = OrderStatus.PENDING if int(order_id) > 0 else OrderStatus.CANCELLED
```